Inventory.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Inventory : MonoBehaviour
+{
+    public int selectedItemIndex = 0;
+    public InventoryUI inventoryUI;
+    public Item[] items = new Item[5];
+    private int _maxSize = 5;
+    private int _count = 0;
+
+    void Start()
+    {
+        inventoryUI.UpdateUI(items);
+    }
+
+    private void Update()
+    {
+        HandleInput();
+    }
+
+    private void HandleInput()
+    {
+        if (Input.GetKeyDown(KeyCode.Alpha1))
+            EquipItem(0);
+        if (Input.GetKeyDown(KeyCode.Alpha2))
+            EquipItem(1);
+        if (Input.GetKeyDown(KeyCode.Alpha3))
+            EquipItem(2);
+        if (Input.GetKeyDown(KeyCode.Alpha4))
+            EquipItem(3);
+        if (Input.GetKeyDown(KeyCode.Alpha5))
+            EquipItem(4);
```

Comment:
Maybe we could use enum or dictionary of some sort here to match KeyCode with `EquipItem()` function?